system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -326,13 +321,54 @@
 }
 
 .topsite-form {
+  .form-input-container {
+    display: flex;
+    max-width: 492px;
+    margin: 0 auto;
+    padding: 32px;
+
+    .top-site-outer {
+      width: 96px;
```

Comment:
Is this width necessary? Seems like it's something a regular top site link would have / want?

Reply:
I ended up removing this but the problem was caused by the `<a>` having display block.